src/SIM/NonLinSIM.C

```diff
@@ -434,7 +434,7 @@ ConvStatus NonLinSIM::checkConvergence (TimeStep& param)
   if (param.iter > 1 && prevNorm > 0.0 && fabs(norm) > prevNorm*0.1)
     status = SLOW;
 
-  if (msgLevel > 0)
+  if (msgLevel >= -1)
```

Comment:
I would prefer `if (msgLevel > 0 || msgLevel == -1)` such that `msgLevel == 0` still retains silence.

Reply:
done.